pdi-layout-plugin/src/main/java/org/pentaho/community/di/impl/provider/HorizontalLayout.java

```diff
@@ -16,15 +16,16 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
 package org.pentaho.community.di.impl.provider;
 
 
 import com.tinkerpop.blueprints.Graph;
```

Comment:
Can we pass com.tinkerpop.blueprints.Graph into applyLayout? Let the layout manager handle job/trans type conversion, since that should be common